src/blocks/types.ts

```diff
@@ -303,7 +303,58 @@ export enum XClientName {
   javascript = 'Curvenote Javascript Client',
 }
 
-export type Author = { plain: string; user: null } | { user: string; plain: null };
+
+
+export enum WellKnowContributorRoles {
+  Conceptualization = 'Conceptualization',
+  DataCuration = 'Data curation',
+  FormalAnalysis = 'Formal analysis',
+  FundingAcquisition = 'Funding acquisition',
+  Investigation = 'Investigation',
+  Methodology = 'Methodology',
+  ProjectAdministration = 'Project administration',
+  Resources = 'Resources',
+  Software = 'Software',
+  Supervision = 'Supervision',
+  Validation = 'Validation',
+  Visualization = 'Visualization',
+  WritingOriginalDraft = 'Writing – original draft',
+  WritingReviewEditing = 'Writing – review & editing',
```

Comment:
Should be a smaller enum with niceNames that then look up into strings? Or do you think this is fine as is?